src/server.js

```diff
@@ -71,8 +71,14 @@ module.exports = function() {
 
 	manager.sockets = sockets;
 
-	var protocol = config.https.enable ? "https" : "http";
-	log.info("The Lounge v" + pkg.version + " is now running on", protocol + "://" + (config.host || "*") + ":" + config.port + "/", (config.public ? "in public mode" : "in private mode"));
+	let protocol = config.https.enable ? "https" : "http";
+	let host = config.host || "*";
+	if (Helper.getGitCommit()) {
+		log.info(`The Lounge is running from ${Helper.getVersion()}`);
+	} else {
+		log.info(`The Lounge ${Helper.getVersion()} is now running`);
```

Comment:
Not a particular fan of using the ES6 bollocks. The `log` allows for concat by doing:
`log.info("The Lounge", Helper.getVersion(), "is now running");`